packages/@aws-cdk/aws-apigateway/test/test.restapi.ts

```diff
@@ -1051,4 +1052,57 @@ export = {
       test.done();
     },
   },
+
+  '"addToPolicy" adds new policy statement'(test: Test) {
+    // GIVEN
+    const stack = new Stack();
+
+    // WHEN
+    const api = new apigw.RestApi(stack, 'api');
```

Comment:
Update this test case so that a policy statement is specified in the `RestApi` constructor and via `addToPolicy()` and assert that both are present in the final CF resource.